src/dotnet/commands/dotnet-help/HelpCommand.cs

```diff
@@ -25,7 +25,7 @@ public class HelpCommand
   build         Builds a .NET project
   publish       Publishes a .NET project for deployment (including the runtime)
```

Comment:
The order of these seems wrong. Can we alphebatize or put these in order of workflow? Publish is the outlier in that sort order.